pkgs/desktops/gnome/extensions/pano/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib
+, stdenv
+, atk
+, cogl
+, fetchFromGitHub
+, glib
+, gnome
+, gobject-introspection
+, gsound
+, gtk3
+, libgda
+, mkYarnModules
+, nodePackages
+, pango
+, substituteAll
+, yarn
+}:
+
+let
+  version = "10";
+  uuid = "pano@elhan.io";
+  pname = "gnome-shell-extension-pano";
+
+  girpathsPatch = (substituteAll {
+    src = ./set-dirpaths.patch;
+    gda_path = "${libgda}/lib/girepository-1.0";
+    gsound_path = "${gsound}/lib/girepository-1.0";
+  });
+in
+stdenv.mkDerivation rec {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "oae";
+    repo = "gnome-shell-pano";
+    rev = "v${version}";
+    hash = "sha256-1vGiWSXlQ8xheqJsZm3uXCixuLl5NFM2OgPQrzCPhME=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    nodePackages.rollup
+    nodePackages.yarn
+  ];
+
+  buildInputs = [
+    atk
+    cogl
+    glib
+    gnome.gnome-shell
+    gnome.mutter
+    gsound
+    gtk3
+    libgda
+    pango
+  ];
+
+  nodeModules =
+    mkYarnModules {
+      inherit pname version;
+      name = "${pname}-modules-${version}";
+      packageJSON = "${src}/package.json";
+      yarnLock = "${src}/yarn.lock";
+      yarnNix = ./yarn.nix;
+    };
+
+  postPatch = ''
+    substituteInPlace resources/metadata.json \
+      --replace '"version": 999' '"version": ${version}'
+  '';
+
+  buildPhase =
+    let
+      dataDirPaths = lib.concatStringsSep ":" [
+        "${gnome.gnome-shell}/share/gnome-shell"
+        "${gnome.mutter}/lib/mutter-10"
+        "${libgda}/share/gir-1.0"
```

Comment:
This is also a standard path so I would expect it to be picked up from `buildInputs`.

Reply:
You are right. Removed.